src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+
+```
+Revolutionary Major Victory! [Green text]
+X% of Crew were converted to the Revolution. [Yellow text]
+X% of Command were converted to the Revolution.  [Green text]
+0 Unconverted Command made it to CentComm. [Green text]
+X Head Revolutionaries made it to CentComm. [Green text if >0]
+```
+
+```
+Revolutionary Minor Victory! [Green text]
+X% of Crew were converted to the Revolution. [Yellow text]
+X% of Command were converted to the Revolution. [Red text]
+0 Unconverted Command made it to CentComm. [Green text]
+X Head Revolutionaries made it to CentComm. [Green text if >0]
+```
+
+```
+Revolutionary-Crew Tie! [Green text]
+X% of Crew were converted to the Revolution. [Yellow text]
+X% of Command were converted to the Revolution. [Red text]
+0 Unconverted Command made it to CentComm. [Red text]
+0 Head Revolutionaries made it to CentComm. [Red text]
+```
+
+```
+Crew Minor Victory! [Green text]
+X% of Crew were converted to the Revolution. [Green text]
+X% of Command were converted to the Revolution. [Green text]
+X Loyal Command made it to CentComm. [Red text]
+0 Head Revolutionaries made it to CentComm. [Red text]
+```
+
+```
+Crew Major Victory! [Green text]
+All Revolutionaries were killed or deconverted! [Red text]
+```
+
+## Conversion
+
+Conversion is how the Revolutionary antag faction grows, and is a *requirement* to obtain Revolutionary Major Victory. To convert a crewmember they must be interacted with by a Revolutionary over time, and once a critera is met they can be converted to the Revolution. Crew are also able to de-convert Revolutionaries. Conversion is not restricted to Headrevs, though they have access to certain tools that benefit them in this regard.
+
+To represent the convertable state a crewmember is in and the faction they belong to, there is now a hidden nummerical stat called Loyalty Health Points (LHP). 
+
+### Loyalty & Loyalty Damage
+
+All sentient crew have Conversion Health Points, which can normally range between 100 to -100 LHP. All crew, unless specified, begin at a full 100 LHP.
+
+- Revolutionaries are able to do specific actions that "deal damage" to a crewmember's LHP, gradually lowering it with varying effectiveness depending on how covert or obviously antagonistic the action is. 
+- If a crewmember gets below <**10** LHP, they enter a Convertable state.
+  - As long as the crewmember is not a Revolutionary, their LHP can not be reduced below 0. They will remain in the Convertable state until their LHP rises above **10** LHP again.
+  - While a crewmember is in the Convertable state, they can be converted into a Revolutionary via specific actions available to Revolutionaries.
+  - When converted to a Revolutionary, their LHP is set to **-25**.
+  - A successful conversion is silent for surrounding players, however the player who is converted gets a pop-up and the appropriate music stinger.
+- Crew are able to "heal" LHP via specific actions, deconverting Revolutionaries into normal crewmembers.
```

Comment:
It needs to be hard/impossible for a Rev to do this themselves, as otherwise carebear players will metagame themselves back to the crew's side.

Remember, Revs do not have to be players who have HR ticked, and could be a newbie or a hardcore bar RPer who hates all conversion modes. "Discovered" rounds often lead to people cryo-ing. The mode needs to work for players who don't want to be revs because it's not what they logged onto SS14 to do.

Reply:
Appreciate this is super hard to get right, but "oh, I got made into a rev, time to go hug some secoffs and get whacked back into loyalty" or whatever is a problem to solve.

Reply:
A rev player who doesn't want to play rev will always find a way to not play; whether that be through suiciding, cryoing or blatant deconversion. I am willing to try out harder coversions, but I'd rather mitigate it through other means (e.g. not being able to tell who is headrev).